nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/resources/OSGI-INF/disable-removing-comment-children-listener.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.ecm.platform.comment.service.listener.test"
+           version="1.0">
+
+  <extension target="org.nuxeo.ecm.core.event.EventServiceComponent"
+             point="listener">
+    <listener name="commentListener" enabled="false" />
```

Comment:
Not sure why you need to disable this listener?

Reply:
disabling the removed listener is required otherwise, removing the parent will remove the comments and we will ends with testing nothing (i mean comment with removed parent)

Reply:
That's `docRemovedCommentListener` right? `commentListener` is triggered when removing a comment AFAIK.

Reply:
ah let me check about it, my idea was to disable the two to be consistent with existing listener (removed doc, removed comment)